Makefile

```diff
@@ -682,7 +682,6 @@ $(addprefix .build-artefacts/annotated/, $(SRC_JS_FILES) src/TemplateCacheModule
 ${PYTHON_VENV}:
 	mkdir -p .build-artefacts
 	virtualenv --no-site-packages $@
-	${PIP_CMD} install -U pip
```

Comment:
I think it's related to the new ways flake8 is managing extensions since version 3.0.0 (they are to installed per default)
I observed the issue before updating pip.
flake8==2.6.2
solves the issue